tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/image_block.html

```diff
@@ -1,12 +1,22 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 <div class="image grid__image">
-    {% if value.image_is_decorative %}
-        {% srcset_image value.image format-webp saturation-0.6 width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" alt="" %}
-    {% elif value.alt_text %}
-        {% srcset_image value.image format-webp saturation-0.6 width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" alt=value.alt_text %}
+    {% if value.desaturate_image %}
+        {% if value.image_is_decorative %}
+            {% srcset_image value.image format-webp saturation-0.6 width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" alt="" %}
+        {% elif value.alt_text %}
+            {% srcset_image value.image format-webp saturation-0.6 width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" alt=value.alt_text %}
+        {% else %}
+            {% srcset_image value.image format-webp saturation-0.6 width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" %}
+        {% endif %}
     {% else %}
-        {% srcset_image value.image format-webp saturation-0.6 width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" %}
+        {% if value.image_is_decorative %}
+            {% srcset_image value.image format-webp width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" alt="" %}
+        {% elif value.alt_text %}
+            {% srcset_image value.image format-webp width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" alt=value.alt_text %}
+        {% else %}
+            {% srcset_image value.image format-webp width-{400,900,1800} sizes="(max-width: 1022px) 90vw, (max-width: 1879px) 50vw, (min-width: 1880px) 910px" loading="lazy" %}
+        {% endif %}
```

Comment:
Thank you for this! 

This might probably require a bit more time to implement correctly, ensuring that the changes don't break the pattern library. Will add this to the tech debt backlong and revisit it later